### PR TITLE
Added: Alert component and changes in ui

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Alert component added.
- changes in main ui of the component library.
- Live link for the PR --> https://component-demo.netlify.app/pages/docs.html
- not added responsiveness

### CODE REVIEW

File: styles/_components.css [hunk 1]
```diff
@@ -94,3 +90,70 @@
...
+    background-color: #2b3e50;
+    overflow: auto;
+    border-radius: 1rem;
+}
+
+.iframe {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+    /* border: 1px solid var(--color-background); */
```
Comment (DivyanshSareen) [inline styles/_components.css]: remove comment if not intended for future reference

File: styles/_layout.css [hunk 2]
```diff
@@ -34,17 +34,11 @@ body {
 
 .flex-row {
     display: flex;
-    /* justify-content: center;
-    align-items: center; */
-    /* flex-wrap: wrap; */
```
Comment (DivyanshSareen) [inline styles/_layout.css:39]: remove extra comments

File: styles/components/alert.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    width: max-content;
+    padding: 0.6rem 1.4rem 0.8rem 1.2rem;
+    border-radius: 0.5rem;
+    display: flex;
+    align-items: center;
+    color: var(--color-primary-dark);
+}
+
+.alert-success {
+    background-color: #bbf7d0;
```
Comment (DivyanshSareen) [inline styles/components/alert.css]: use var for color values

File: styles/_components.css (hunk 1 above)
Comment (pratik-kumar-sahu) [reply styles/_components.css]: sure in final merge I'll remove the whole comments

File: styles/components/alert.css (hunk 3 above)
Comment (pratik-kumar-sahu) [reply styles/components/alert.css]: thanks will change these

**Pull request merged**